deno.jsonc

```diff
@@ -4,9 +4,9 @@
     "packages/ajv-validator",
     "packages/arktype-validator",
     // "packages/auth-js",
-    "packages/bun-compress",
+    // "packages/bun-compress",
```

Comment:
> Unknown export './utils/compress' for '@hono/hono@4.8.3'

Looks like hono is missing an export for `./utils/compress`. There is one for `./utils/*`, but I think Deno/JSR don't understand the wildcard